### PR TITLE
integrating_experiments_online

### PR DESCRIPTION
**Pull request by johanobandoc**

### CODE REVIEW

File: epsilons/main_online_experiments.py [hunk 1]
```diff
@@ -28,19 +28,35 @@
 
 flags.DEFINE_integer("initial_seed", "1", "the program will run seeds [initial_seed, initial_seed + 5)")
 
+flags.DEFINE_string("experiment", "1", "the experiment will be run in")
```
Comment (joaogui1) [inline epsilons/main_online_experiments.py]: I don't understand what this flag does

File: epsilons/main_online_experiments.py [hunk 2]
```diff
@@ -28,19 +28,35 @@
...
+        "inits":inits
+        "activations":activations
+}
+    
+
 num_runs = 1  #7
 # `path=os.environ['AIP_TENSORBOARD_LOG_DIR']`
 
-path = "../../ExperimentsInitializer/epsilons"  #TODO point to cloud bucket
+path = "../../ExperimentsInitializer/epsilons"  #TODO
```
Comment (joaogui1) [inline epsilons/main_online_experiments.py:49]: don't remove this please, we'll need for xmanager

File: epsilons/main_online_experiments.py [hunk 3]
```diff
@@ -28,19 +28,35 @@
...
+flags.DEFINE_string("experiment", "1", "the experiment will be run in")
+
+flags.DEFINE_boolean("weights_biases", "False", "the program won't use weights&biases")
+
+experiments = {
+        "epsilons": epsilons,
+        "learning_rates":learning_rates,
+        "widths":widths,
+        "depths":depths,
+        "normalizations":normalizations
```
Comment (joaogui1) [inline epsilons/main_online_experiments.py]: missing commas

File: epsilons/main_online_experiments.py [hunk 4]
```diff
@@ -49,24 +65,50 @@ def create_agent(sess, environment, summary_writer=None): #memory=None
                                 "random seed": i,
                                 "agent": FLAGS.agent,
                                 "environment": FLAGS.env,
-                                "epsilon": eps, 
-                                "varying": "epsilon",
+                                f'{FLAGS.exp}': eps,
```
Comment (joaogui1) [inline epsilons/main_online_experiments.py:68]: Cool idea!

File: epsilons/main_online_experiments.py (hunk 1 above)
Comment (joaogui1) [reply epsilons/main_online_experiments.py]: explain it a little more please

Comment (johanobandoc) [reply epsilons/main_online_experiments.py]: This flag chooses which experiment to run. For example, epsilon, learning rate, and so on.

File: epsilons/main_online_experiments.py (hunk 3 above)
Comment (johanobandoc) [reply epsilons/main_online_experiments.py]: Thanks!

File: epsilons/main_online_experiments.py (hunk 1 above)
Comment (johanobandoc) [reply epsilons/main_online_experiments.py]: it should be: 
flags.DEFINE_string("experiment", "epsilons", "the experiment will be run in")

Comment (joaogui1) [reply epsilons/main_online_experiments.py]: Then please update it and push again
